src/live/index.js

```diff
@@ -18,13 +19,15 @@ const onConnect = (io, socket) => {
       `received message from socket: ${socket.id}. Will join room: ${chatRoomName}`
     );
     socket.join(chatRoomName);
+    //console.log(`User joined album chat room for album: ${albumId}`);
   });
 
-  socket.on('chat:album', (data) => {
-    console.log(
-      `received message from socket: ${socket.id}. Message=${data.message} | Album=${data.albumId} | User=${data.userId}`
-    );
-    handleAlbumChat(io, socket, data);
+  socket.on('chat:album', async (data) => {
+    try {
```

Comment:
Since we already have a try/catch inside `handleAlbumChat` function that is catching any errors that occur and logging them; the additional try/catch here is redundant and can be removed.